e2e/support/properties/scrollbars.js

```diff
@@ -0,0 +1,35 @@
+chai.Assertion.addProperty("scrollableHorizontally", function () {
```

Comment:
> Perhaps we need to extract this helper. I'll remember it next time we need it.

[Following up](https://github.com/metabase/metabase/pull/32119#discussion_r1252690036) and making this reusable.

Reply:
I don't think that extending `chai` with non-standard methods is a good idea. This is hard to reason about. Can this functions be extracted as regular ones? We have many e2e helpers available, and we could add them here.

Reply:
> I don't think that extending chai with non-standard methods is a good idea.

I disagree. It reads really easily in tests (`expect(element).to.be.scrollableHorizontally`) and its definition is easily searchable.

>  This is hard to reason about.

Are custom cypress commands not as hard to reason about?

> We have many e2e helpers available, and we could add them here.

The reason I didn't add them there because there are no generic helpers with assertions. There are only 5 _specific_ helpers with assertions - it does not look like a pattern.

It feels like custom assertions belong somewhere else. [Cypress docs guided](https://docs.cypress.io/guides/references/assertions#Adding-New-Assertions) me to this.

Reply:
We avoid adding new custom cypress commands too

Reply:
Refactored to simple functions: 68b57e21e81e4086c20b8b573aab707777450533